src/utils/validation/schema.js

```diff
@@ -0,0 +1,333 @@
+import Validator from 'fastest-validator';
```

Comment:
It was kind of fun.  I like that we're no longer using our custom validator.  Hopefully at some point we get the data validation happening on the backend though, and this can disappear.